agent/src/test/java/com/teamscale/jacoco/agent/options/AgentOptionsParserTest.java

```diff
@@ -18,6 +19,14 @@ public void notGivingAnyOptionsShouldBeOK() throws Exception {
 		parser.parse(null);
 	}
 
+	@Test
+	public void mustPreserveDefaultExcludes() throws Exception {
+		Assertions.assertThat(parser.parse("").jacocoExcludes).isEqualTo(AgentOptions.DEFAULT_EXCLUDES);
+		Assertions.assertThat(parser.parse("excludes=**foo**").jacocoExcludes)
+				.isEqualTo("**foo**:" + AgentOptions.DEFAULT_EXCLUDES);
+		parser.parse(null);
```

Comment:
What does this statement ensure? If it is intentional a comment would be helpful